packages/cloud-cognitive/src/components/UserProfileImage/UserProfileImage.js

```diff
@@ -55,14 +56,14 @@ export let UserProfileImage = React.forwardRef(
         sm: User16,
         md: User20,
         lg: User24,
-        xl: User32,
+        xlg: User32,
       },
       group: {
         xs: Group16,
         sm: Group16,
         md: Group20,
         lg: Group24,
-        xl: Group32,
+        xlg: Group32,
```

Comment:
Is having an `xs` icon necessary since it's the same as the small? Probably a design decision but now I'm curious 🤔